### PR TITLE
[BH-1191] Wire posts to posts lib

### PR DESCRIPTION
**Pull request by ChrisWiegman**
## Description

This wires the acm content library to the plugin to allow for saving and editing relationships in a much more robust platform.

https://wpengine.atlassian.net/browse/BH-1191

### CODE REVIEW

File: includes/publisher/class-publisher-form-editing-experience.php [hunk 1]
```diff
@@ -296,13 +301,22 @@ public function save_post( int $post_id, WP_Post $post ): void {
 		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
 		$posted_values = $_POST['atlas-content-modeler'][ $post->post_type ];
 
+		$saved_replationships = array();
```
Comment (nickcernis) [inline includes/publisher/class-publisher-form-editing-experience.php:304]: Typo in `$saved_replationships`.

Comment (ChrisWiegman) [reply includes/publisher/class-publisher-form-editing-experience.php]: Good catch, Nick! This has been fixed.

**Review by nickcernis — APPROVED** [review]
This looks good. I confirmed that the relationship table is populated on save, and that relationships are moved when one-to-many relationships are unchecked.

Screenshot 2021-08-24 at 17 50 16

**Review by mindctrl — APPROVED** [review]
Looking good here. Confirmed the data is stored and retrieved as planned. Locally I merged this into the branch for #222 and it all appears to be working well together.

**Pull request merged**